examples/2_advanced/plot_s4_galaxies.py

```diff
@@ -0,0 +1,166 @@
+'''
+Stage IV Space Satellite Galaxy Survey
+======================================
+
```

Comment:
Maybe just "S4 Galaxy Survey" for brevity, and then mention both hemispheres/space telescope in the text?

Reply:
Changed it to:
```
SIV Galaxy Survey
==============
```